### PR TITLE
refactor: Update Argo Workflow with EKS Blueprints V5 and new EKS Data Addons

### PR DESCRIPTION
**Pull request by ovaleanu**
### What does this PR do?

🛑 Please open an issue first to discuss any significant work and flesh out details/direction - we would hate for your time to be wasted.
Consult the [CONTRIBUTING](https://github.com/awslabs/data-on-eks/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests.

- Update for using EKS Blueprints V5
- Update Prometheus to kube-prometheus-stack
- Use Argo Events addon from EKS Bluprints Addons
- Use EKS Data Addons repo

### More

- [x] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ ] Mandatory for new blueprints. Yes, I have added a example to support my blueprint PR
- [ ] Mandatory for new blueprints. Yes, I have updated the `website/docs` or `website/blog` section for this feature
- [x] Yes, I ran `pre-commit run -a` with this PR. Link for installing [pre-commit](https://pre-commit.com/) locally

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: schedulers/terraform/argo-workflow/argo-events-manifests/eventbus.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#            cpu: "10m"
+#      metricsContainerTemplate:
+#        resources:
+#          requests:
+#            cpu: "10m"
+#      antiAffinity: false
+#      persistence:
+#        storageClassName: standard
+#        accessMode: ReadWriteOnce
+#        volumeSize: 10Gi
```
Comment (vara-bonthu) [inline schedulers/terraform/argo-workflow/argo-events-manifests/eventbus.yaml:25]: Do we need this to be commented?

File: schedulers/terraform/argo-workflow/argo-events-manifests/eventsource-sqs.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+spec:
+  template:
+    serviceAccountName: event-sa
+  sqs:
+    example:
+      # jsonBody specifies that all event body payload coming from this
+      # source will be JSON
+      jsonBody: true
+      # aws region
+      region: us-east-1
```
Comment (vara-bonthu) [inline schedulers/terraform/argo-workflow/argo-events-manifests/eventsource-sqs.yaml:15]: region is hardcoded. Is there a way we can allow customers to change the region from the TF code?

**Review by vara-bonthu — APPROVED** [review]
LGTM 🚀

File: schedulers/terraform/argo-workflow/argo-events-manifests/eventsource-sqs.yaml (hunk 2 above)
Comment (ovaleanu) [reply schedulers/terraform/argo-workflow/argo-events-manifests/eventsource-sqs.yaml]: I modified the region and queue name to be as variable.

File: schedulers/terraform/argo-workflow/argo-events-manifests/eventbus.yaml (hunk 1 above)
Comment (ovaleanu) [reply schedulers/terraform/argo-workflow/argo-events-manifests/eventbus.yaml]: I think we should leave to see what options can have. I corrected the storageClassName though.

**Review by vara-bonthu — APPROVED** [review]
LGTM 🚀

**Pull request merged**